README.md

```diff
@@ -6,7 +6,11 @@ The ESLint generator for [Yeoman](https://yeoman.io/). This generator is intende
 
 First and foremost, you must have [Node.js](https://nodejs.org/) and npm installed. If you don't have Node.js installed, please download and install the latest version.
 
-You must also install Yeoman, if you don't have it installed already. To install Yeoman, you can run this command:
+> **Requirements**
+>
+> - Node.js v14.17.x, v16.x and >=18.0.0
+
+You must also install Yeoman, if you don't have it installed already.To install Yeoman, you can run this command:
```

Comment:
Looks like this line shouldn't be touched.

Reply:
Yes but I had to move it since I wanted the requirements to come after the node installment message

Reply:
it's missing a space between sentences now

Reply:
Okay. Done

Reply:
The space is still missing.

Reply:
Done.